packages/sysweb3-keyring/src/keyring-manager.ts

```diff
@@ -397,8 +397,16 @@ export class KeyringManager implements IKeyringManager {
     ) {
       throw new Error('Cannot remove active network');
     }
-    delete this.wallet.networks[chain][chainId];
+    // Create a new object without the specified property
+    const updatedNetworks = Object.fromEntries(
+      Object.entries(this.wallet.networks[chain]).filter(
+        ([key]) => Number(key) !== chainId
+      )
+    );
+    // Replace the networks object for the chain with the updated object
+    this.wallet.networks[chain] = updatedNetworks;
```

Comment:
We're having a typeError on the test: Cannot assign to read only property 'ethereum' of object. Related to this line where we try to overwrite the ethereum networks